API_TRANSITION_GUIDE.md

```diff
@@ -0,0 +1,62 @@
+## Deprecated API Transition Guide
+
+When API's are deprecated, the following guide will show how to transition from removed API's to the new ones
+
+
+### Upgrading From 3.x to 4.0
+
+In the 3.2.0 release, a few API's were set to be deprecated in the 4.0 release.  With the release of 4.0.0,  those API's have been removed.
+
+#### Receiever
+
+The `Receiver` class has been removed.
+
+`Receiver.accept` should be transitioned to `HTTP.toEvent`
+
+#### Emitter
+
+The `Emitter` class has been removed.
```

Comment:
Given #366 perhaps something like
```suggestion
The `Emitter` class has been changed with the removal of `Emitter.send()` and the addition of `Emitter.emit()`. This change puts the responsibility for network transport on the developer. See the example below.
```

It might be good to get @loopingz to contribute some text around `Emitter.emit()` for this guide as well.

Reply:
:+1: